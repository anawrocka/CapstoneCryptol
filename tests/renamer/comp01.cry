module comp01 where

// Test that comprehensions are getting renamed properly
xs = [ a | a <- [1 .. 10] ]
