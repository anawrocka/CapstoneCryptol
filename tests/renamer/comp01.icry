:l comp01.cry
