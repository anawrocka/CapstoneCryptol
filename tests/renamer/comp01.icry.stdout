Loading module Cryptol
Loading module Cryptol
Loading module comp01
